### ISSUE TITLE
Feature Request: Show/Hide Menubar

### ISSUE THREAD

**Issue by dustigroove** [nylas/N1#1181]
Per the inquiry and interest shown in issue #304;  Would it be possible to add the ability to show/hide the  menu bar (a la Firefox, Atom, Nemo, Windows Explorer, etc, etc.) by pressing the Alt key? This would greatly help those of us that aren't mouse tethered to reclaim a substantial amount of screen real estate and would make N1 look more seamless.

With menu bar:
![n1_with_menubar](https://cloud.githubusercontent.com/assets/3325820/12725944/1ca4b2b2-c8e3-11e5-8beb-fde0d408f625.png)

Without menu bar:
![n1_no_menubar](https://cloud.githubusercontent.com/assets/3325820/12725965/2cac2140-c8e3-11e5-9598-946e4e8f8a5d.png)

**Comment by fabianalexisinostroza** [comment]
pleade add these feature ... i'm using nylas with Plasma 5 desktop and think is possible add a menu button for access to configurations

**Comment by brittyazel** [comment]
I agree, using N1 with any non-Unity desktop leaves the UI feeling a bit off.

**Comment by felipetavares** [comment]
Some relevant links:

* [Electron's implementation](https://github.com/atom/electron/pull/558)
* [Atom relevant issue](https://github.com/atom/atom/issues/2914)
* [Another one](https://github.com/atom/electron/issues/486)

**Comment by varlesh** [comment]
+1

**Comment by alexbooker** [comment]
Just writing to express some support for this feature request.

**Comment by martinring** [comment]
+1
The menu interrupts the ui and looks ugly

**Comment by G-Ray** [comment]
+1

**Comment by rhoconlinux** [comment]
+1

**Comment by AdnanHodzic** [comment]
+1

It would also be nice if on GNOME, menu bar would be moved to become part of GNOME GTK/Headerbar.

**Comment by chmln** [comment]
+1

**Comment by alisonbnt** [comment]
+1

**Comment by brittyazel** [comment]
I would be happy with either an "alt to show" solution to this problem, or just the "cogwheel" button on the right hand side that is present in the Windows versions of N1. Anything but the ugly menu bar. And with the new dark themes they included, it's like they are teasing us ;-)

**Comment by sreevisakh** [comment]
+1

**Comment by NeoMahler** [comment]
+1000!

**Comment by topaway** [comment]
+1

**Comment by PaNaVTEC** [comment]
+1

**Comment by lifow** [comment]
+1

**Comment by michaelplews** [comment]
+1

**Comment by nobol** [comment]
+1

**Comment by ozky** [comment]
Would be nice to lxqt too maybe in future add support to use other than gnome-keyring too lxqt wallet and kwallet. 
Name lies bit as it supports gnome keyring and kwallet backend too.
https://github.com/mhogomchungu/lxqt_wallet

**Comment by madbessoul** [comment]
+1

**Comment by ericdodds** [comment]
+1
